packages/contracts-bedrock/deploy/020-SystemDictatorSteps-1.ts

```diff
@@ -73,10 +71,13 @@ const deployFn: DeployFunction = async (hre) => {
 
   // Transfer ownership of the ProxyAdmin to the SystemDictator.
   if ((await ProxyAdmin.owner()) !== SystemDictator.address) {
-    console.log(`Setting ProxyAdmin owner to MSD`)
-    await ProxyAdmin.transferOwnership(SystemDictator.address)
-  } else {
-    console.log(`Proxy admin already owned by MSD`)
+    await doOwnershipTransfer({
+      isLiveDeployer,
+      proxy: ProxyAdmin,
+      name: 'ProxyAdmin',
+      transferFunc: 'transferOwnership',
+      dictator: SystemDictator,
+    })
```

Comment:
I recognize that the proxy admin is always owned by the `deployer` key as it is configured that way in the deployment scripts, so this doesn't use all the functionality of the new function, but I still think its cleaner.